vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/AbstractGridMultiSelectionModel.java

```diff
@@ -365,7 +365,7 @@ public boolean isSelectionColumnFrozen() {
     protected abstract void fireSelectionEvent(
             SelectionEvent<Grid<T>, T> event);
 
-    void clientSelectAll() {
+    protected void clientSelectAll() {
```

Comment:
We might as well expose `clientDeselectAll` for consistency.

Reply:
Done